### ISSUE TITLE
Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead

### ISSUE THREAD

**Issue by jahirfiquitiva** [junkboy0315/react-compare-image#102]
I'm getting this error with Next.js `13.3.0`, React `18.2.0` and `react-compare-image` `3.3.0`

```gradle
Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead
```

**Comment by jahirfiquitiva** [comment]
Just noticed there's an open PR for this: https://github.com/junkboy0315/react-compare-image/pull/100

Please merge it soon @junkboy0315

**Comment by junkboy0315** [comment]
released as v3.4.0!
